src/main/java/duke/commands/AddCommand.java

```diff
@@ -34,6 +35,8 @@ public AddCommand(String[] components, String fullCommand) throws DukeException
             throw new InputException("☹ OOPS!!! The description of a deadline cannot be empty.");
         } else if (this.type.equals("event") && components[1].equals("/by")) {
             throw new InputException("☹ OOPS!!! The description of an event cannot be empty.");
+        } else if (this.type.equals("event") && components[1].equals("/needs")) {
```

Comment:
Please fix the bug here.

Reply:
Thanks for spotting this trivial mistake!